### ISSUE TITLE
Audio ducking is not working correctly in win10 version1703

### ISSUE THREAD

**Issue by abdullah5490** [nvaccess/nvda#7069]
Hi NVDA developers. It is to aware you that the audio ducking feature is conflicting in win10 latest version 1703. The sounds of all apps always be ducked even the option is set to "no ducking". We need to first change the option to "duck when outputting speech", and then again open the dialog and set it back to "no ducking" and save it, to allow other apps to output their sounds loudly. And This option change is needed on every restart of NVDA, otherwise there will be no louder sound from any application. This bug is only related to v1703 of the win10, and i'm using NVDA 2017.1. Please provide an update to resolve it, or at least, tell me any way to remove this ducking feature temprarily in my installation of NVDA

**Comment by jcsteh** [comment]
This is not something I've seen unless Narrator was running at some point. Do you ever use Narrator? If so, it may be causing this. To confirm that, please ensure Narrator is not set to start at or after logon, restart your system and start NVDA after the system restarts. Then, report whether the problem persists. Thanks.

**Comment by fernando-jose-silva** [comment]
I had a similar problem when trying to use the AudioThemes3D-5.1 addon
When the addon was enabled all sounds would get lower, even when the nvda was set to redo when playing speech and sounds activated.
With the priority audio setting turned off there were no problems.
This addon also causes an error in the nvda when navigating the settings of the cortana (setting on me).

**Comment by jage9** [comment]
If you run Narrator while NVDA is running, ducking persists until you restart NVDA, regardless of the NVDA setting. I'm not sure if we should check more often than on restart to apply the setting, but the best solution may be to also turn off ducking in Narrator if it is a feature you don't want anyway. There probably aren't a lot of use cases where you would want it enabled in Narrator but not NVDA.
TL;DR: Probably quicker to just restart NVDA when this happens instead of adding a bunch of code to persistently check for the existence of ducking.

**Comment by jcsteh** [comment]
We actually can't "check" the state of ducking; we can only set it. The fact that other apps interfere even after they exit is simply a Windows bug that needs to be fixed by Microsoft. We've notified Microsoft about this.

CC @MichaelDCurran.

**Comment by michaelDCurran** [comment]
Reported MS Bug 11789608 for this.

**Comment by Adriani90** [comment]
@abdullah5490, @jage9 is this issue still occuring in Windows 10 1809 Update?

**Comment by abdullah5490** [comment]
no, its working fine.

**Comment by Adriani90** [comment]
Closing as works for me. @abdullah5490     thank you for your promt reaction.

**Comment by BlindMerican** [comment]
this is still going on. even if all ducking in nvda is disabled AND win 10 sound enhancements are set to "do nothing" AND all apps are set to volume 100% AND even if the fade on nvda is set to value of "0" as soon as ANY communication app such as discord or signal send any media there it goes again. There is a bug with nvda and win 10's ducking. Having to restart the computer every few minutes is NOT a solution. contacting microsoft only provides the response that it is an nvda, according to microsoft. the accessibility dept refused to even open a tickt. this has obviously been a problem for years and yet, when i emailed nvda support they claimned they had never heard of this problem. there are NUMEROUS search results using ANY browser that pop up this problem and yet it does NOT occur with JAWS. One would think , if it were a win 10 only but, that it would occur across the board with JAWS as well, ut it DOESNT. I gues the only choice is to get rid of NVDA and pay for JAWS and advise everyone I know to do that as well. Audio ducking isnt the only problem with nvda. the mouse tracking is problematic , also, making it worthless for those of us still learning keyboard commands. Shame. That's the whole reason I chose nvda over jasws, but no reason to if nvda cant make their program work

**Comment by XLTechie** [comment]
@BlindMerican as you can read from the above comments, this was fixed for Windows 10 in or before 2018. That's over six years ago.

If you're having this problem on a modern version of NVDA and a modern version of Windows, it is likely a different cause.
Please open a new issue, and fill out the requested information on the template.

That will enable people to debug it properly, once you list the software versions, etc.

This issue is about a 2017 version of Windows, and a 2017 version of NVDA, both of which are entirely outdated and irrelevant to anything running today.

**Comment by BlindMerican** [comment]
your template makes no sense. i opened account spicifically to get help, sinse contacting nvda email support got me nowhere. all your template shows in "new issue" is a bunch of ####'s and >'s that a newly introduced NVDA user with no background in programming wouldnt know. NVDA has made it as difficult as they can for someone newly blind to report any bug or get help regarding one
